server/chat-plugins/chat-monitor.js

```diff
@@ -296,17 +296,13 @@ let nicknamefilter = function (name, user) {
 	return name;
 };
 
-/** @typedef {(query: string[], user: User, connection: Connection) => (string | null | void)} PageHandler */
-/** @typedef {{[k: string]: PageHandler | PageTable}} PageTable */
-
 /** @type {PageTable} */
 const pages = {
-	filters(query, user, connection) {
+	async filters(query, user, connection) {
```

Comment:
Why async?

Reply:
Was to test whether async worked but this can go now forgot to do that